scene/resources/environment.cpp

```diff
@@ -694,150 +694,59 @@ bool Environment::is_fog_enabled() const {
 	return fog_enabled;
 }
 
-void Environment::set_fog_color(const Color &p_color) {
-	fog_color = p_color;
+void Environment::set_fog_light_color(const Color &p_light_color) {
+	fog_light_color = p_light_color;
 	_update_fog();
 }
-
```

Comment:
@reduz You're doing it again :)

Our style guide mandates a separate line between all methods, and `clang-format` can't enforce it, so you need to change your habit.

Reply:
I do my best, but I generally write some much code that I forget sometimes. I think until Clang Format handles this properly it will be difficult to keep consistency.